frappe/translations/fr.csv

```diff
@@ -4852,3 +4852,4 @@ Published on,Publié le,
 Bottom,Bas,
 Top,Haut,
 List View,Vue en liste,
+Edit:edit-grid-row,Détail
```

Comment:
```suggestion
Edit,Détail,Edit grid row,
```

Context is a separate column. Concatenating using `:` is only done in code internally.